src/osemgrep/language_server/server/Session.ml

```diff
@@ -180,7 +186,15 @@ let fetch_rules session =
       rules
   in
   let rule_filtering_conf =
-    Rule_filtering.{ exclude_rule_ids = []; severity = [] }
+    Rule_filtering.
+      {
+        exclude_rule_ids = [];
+        severity = [];
+        exclude_products =
+          [
```

Comment:
maybe add a comment here explaining why those products are excluded
(I guess because LSP.js does not support pro yet, and osemgrep does not support SCA and Secrets yet).